samples/client-provider/query.fsx

```diff
@@ -0,0 +1,37 @@
+﻿#r "../../src/FSharp.Data.GraphQL.Client/bin/Debug/FSharp.Data.GraphQL.Client.dll"
+
+open FSharp.Data.GraphQL
+open System.Collections.Generic
+
+let [<Literal>] serverUrl = "http://localhost:8083"
+
+// The name and arguments of the query will be automatically set by the type provider
+let [<Literal>] queryFields = "{ id, name, appearsIn, friends { name } }"
+let [<Literal>] queryFieldsWithFragments = "{ ...data, friends { name } } fragment data on Human { id, name, appearsIn }"
+
+type MyClient = GraphQLProvider<serverUrl>
+
+let hero =
+    MyClient.QueryHero<queryFieldsWithFragments>("1000")
```

Comment:
You're right, there should also be a generic `Query` method, I'll add it :+1: Thanks!